CHANGELOG.md

```diff
@@ -1,4 +1,9 @@
+# 2.0.0
```

Comment:
We should add some guidance for upgrading from 1.1 to 2.0. It can go in the README, e.g. what changes do they need to make.

This changelog should also list what those breaking changes are.